mingw-w64-duktape/PKGBUILD

```
@@ -0,0 +1,50 @@
+_realname=duktape
+pkgbase=mingw-w64-${_pkgname}
+pkgname="${MINGW_PACKAGE_PREFIX}-duktape"
+pkgver=2.7.0
+_dirname=${_realname}-${pkgver}
+pkgrel=1
+url="http://duktape.org/"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+pkgdesc="Embeddable Javascript engine"
+license=("MIT")
+depends=("${MINGW_PACKAGE_PREFIX}-crt-git")
+makedepends=("${MINGW_PACKAGE_PREFIX}-gcc")
+options=(staticlibs !strip !buildflags)
+source=(
+	"http://duktape.org/duktape-${pkgver}.tar.xz"
+	"duktape.pc"
+	)
+sha256sums=('90f8d2fa8b5567c6899830ddef2c03f3c27960b11aca222fa17aa7ac613c2890'
+            '5a888e877d3ae2c752aba75a88587c22a4ef96b428a53f7b9b435799c758a55a')
+
+_architectures="${MINGW_PACKAGE_PREFIX}"
+
+build() {
+  for _arch in ${_architectures}; do
+    mkdir -p "${srcdir}/${pkgname}-${pkgver}-build-${_arch}"
+    pushd "${srcdir}/${_dirname}" > /dev/null
+    gcc src/duktape.c -std=c99 -O2 -c -o duktape.o
```

```suggestion
    cc src/duktape.c -std=c99 -O2 -c -o duktape.o
```

pref usage of `cc` over `gcc` or `clang` or alternatively,

```bash
    ${CC:-cc} src/duktape.c -std=c99 -O2 -c -o duktape.o
```

---

Ok, I really only made this for MinGW64 and then just tested to see if it built for the other environments